lisa/sut_orchestrator/azure/common.py

```diff
@@ -972,6 +972,16 @@ def check_or_create_resource_group(
                 rm_client.resource_groups.create_or_update(
                     resource_group_name, {"location": location}
                 )
+            timeout = 60
+            timeout_timer = create_timer()
+            while timeout_timer.elapsed(False) < timeout:
+                if rm_client.resource_groups.check_existence(resource_group_name):
+                    break
+                sleep(2)
```

Comment:
It's second time to use the timeout checking pattern, please create a method in utils/__init__.py.

Reply:
updated